### ISSUE TITLE
repeatedly pressing down in general and sound settings changes last highlighted option

### ISSUE THREAD

**Issue by briaguya0** [Mr-Wiseguy/Zelda64Recomp#366]
**Describe the bug**

in the general and sound settings menus, pressing down repeatedly on keyboard or controller switches between highlighting the first and second option of the last setting (currently "Analog Camera Mode" and "Low Health Beeps")

**To Reproduce**
1. go to "General" settings
2. repeatedly press down

**Expected behavior**
pressing down when in the last setting's options doesn't do anything

**Screenshots**

[Screencast from 2024-06-05 01-16-09.webm](https://github.com/Zelda64Recomp/Zelda64Recomp/assets/70942617/4b5c51ae-b36c-4450-a6fe-dfdf2b85efef)

[Screencast from 2024-06-05 01-16-57.webm](https://github.com/Zelda64Recomp/Zelda64Recomp/assets/70942617/cc3ff8e7-b93c-4d63-b9dc-774329ce70f3)

**Comment by Mr-Wiseguy** [comment]
Good catch, these should also get fixed by the same refactor that I mention [here](https://github.com/Zelda64Recomp/Zelda64Recomp/issues/367#issuecomment-2151304500).
